routes/markers.js

```diff
@@ -142,9 +142,9 @@ module.exports = function(connection) {
     router.patch('/marker/:id', utils.isAuthenticated, function (req, res, next) {
 
         //If authenticated user is not the owner of the marker or an admin, return 401 (Unauthorized)
-        if(req.session.user.uid != req.body.uid && !req.session.user.admin) return res.send(401);
+        if(req.session.user.uid != req.body.created_by && !req.session.user.admin) return res.send(401);
```

Comment:
We need to check the value of `created_by` in the database and compare to that value.

Reply:
Yup, we're doing that too, in the sql syntax with these lines.
```
if(!req.session.user.admin) {
                query += ' AND created_by=\''+req.session.user.uid+'\'';
}
```

Reply:
Ah yes. Now I remember. We discussed this on a prior PR. I got to my laptop and looked at the full method. Now I see that users can only create a marker with their own username as creator unless they're an admin. That's fine, though it would probably be better not to depend on the client to provide this value.

I just wanted to make sure we weren't taking a value unchecked from the client. This detail doesn't need to hold up this PR. I'll resolve this conversation.